homeassistant/components/xiaomi_miio/strings.json

```diff
@@ -5,7 +5,8 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
       "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
       "incomplete_info": "Incomplete information to setup device, no host or token supplied.",
-      "not_xiaomi_miio": "Device is not (yet) supported by Xiaomi Miio."
+      "not_xiaomi_miio": "Device is not (yet) supported by Xiaomi Miio.",
+      "unknown": "[%key:common::config_flow::error::unknown%], please check the log."
```

Comment:
Usually we don't use partials here since the message can change which make this message nonsensical

Reply:
I removed the `, please check the log.`